global.json

```diff
@@ -1,11 +1,11 @@
 {
   "sdk": {
-    "version": "7.0.100-rc.1.22431.12",
+    "version": "7.0.100",
     "allowPrerelease": true,
```

Comment:
Do we actually want to allow pre-release? that implies that `7.0.100-rc.1` should be chosen over `7.0.100`. I assume we want to disable allowing pre-release until we move to pre-release versions of the .NET 8 SDK.

Reply:
if the rc version is chosen over that then yeah I should definitely remove this.  Didn't know it would do that, will update

Reply:
That would only happen if both `7.0.100` and `7.0.100-rc1` are installed.
I wouldn't block on it.